recipes/cmake/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: cmake
+  version: 3.5.0
+
+source:
+  url: https://cmake.org/files/v3.5/cmake-3.5.0.tar.gz  # [not win]
+  fn: cmake-3.5.0.tar.gz # [not win]
+  
+  url: https://cmake.org/files/v3.5/cmake-3.5.0-win32-x86.zip  # [win]
+  fn: cmake-3.5.0-win32-x86.zip # [win]
+
+
```

Comment:
Nope.  Just nitpicking.  Feel free to ignore, or store up revenge for the next recipe of mine you review.